src/@types/Price.d.ts

```diff
@@ -1,14 +1,53 @@
+import { ProviderFees } from '@oceanprotocol/lib'
+
+/**
+ * @interface OrdePriceAndFee
```

Comment:
```suggestion
 * @interface OrderPriceAndFee
```

Amazing documentation here though 😍